now.json

```diff
@@ -22,13 +20,9 @@
         "cache-control": "public, max-age=43200, immutable"
       }
     },
+    { "handle": "filesystem" },
```

Comment:
@timneutkens what does this line do? I have seen it before, but never in the documentation?

Reply:
@timneutkens @timothyis can one of you point me to the documentation for this^?

Reply:
@mfix22 Here: https://zeit.co/docs/v2/deployments/routes#wildcard-routes

Reply:
Does it matter what order this command is in the routes array though?

Reply:
Yes, it goes over every route 1 by 1